.gitignore

```diff
@@ -1 +1,342 @@
-.DS_Store
\ No newline at end of file
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
+
+##We don't need all these I just copied and pasted a template that I found from googling - Clarence
```

Comment:
Good thinking! These templates are sometimes better to use anyways since a lot of times it's easy to forget about these